<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BulkExporterDialog</class>
 <widget class="QDialog" name="BulkExporterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bulk Exporter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/icons/page_white_go.png</normaloff>:/res/icons/page_white_go.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>The Exporter uses orders listed in the main window, use the filter and search functionality to narrow down which orders you want to export.
The preview is limited to 25 orders for speed reasons, press Export to export all the orders.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="previewTextEdit">
         <property name="font">
          <font>
           <family>Courier New</family>
          </font>
         </property>
         <property name="lineWrapMode">
          <enum>QPlainTextEdit::NoWrap</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="columnGroupBox">
         <property name="title">
          <string>Columns</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTreeWidget" name="columnTreeWidget">
            <property name="indentation">
             <number>0</number>
            </property>
            <attribute name="headerVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string/>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="columnButtonHorizontalLayout">
            <item>
             <widget class="QToolButton" name="columnUpButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/res/icons/arrow_up.png</normaloff>:/res/icons/arrow_up.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="columnDownButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="icon">
               <iconset resource="../res.qrc">
                <normaloff>:/res/icons/arrow_down.png</normaloff>:/res/icons/arrow_down.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="columnHorizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="presetGroupBox">
         <property name="title">
          <string>Presets</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="presetComboBox">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <item>
             <property name="text">
              <string>Save as new preset...</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="deletePresetButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../res.qrc">
              <normaloff>:/res/icons/bin.png</normaloff>:/res/icons/bin.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BulkExporterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BulkExporterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>660</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
